include/nbl/builtin/hlsl/cpp_compat/intrinsics.h

```diff
@@ -71,12 +71,18 @@ inline matrix<T,M,N> transpose(const matrix<T,N,M>& m)
     return glm::transpose(reinterpret_cast<typename matrix<T,N,M>::Base const&>(m));
 }
 
+namespace glsl // TODO ?
+{
```

Comment:
in this case the alias goes into the #else case in this file
https://github.com/Devsh-Graphics-Programming/Nabla/blob/7d90baa1d8009f5c0040b74deb0d4b8f5e3f9584/include/nbl/builtin/hlsl/glsl_compat/core.hlsl#L130